src/js/experiments/snippetFeedback.js

```diff
@@ -42,12 +42,12 @@ export class SnippetFeedback {
     // Create some dom elements:
     const container = this._makeElement({
       kind: 'div',
-      className: 'was-this-helpful',
+      className: 'was-this-code-helpful',
```

Comment:
You have the right idea but let's avoid any markup changes (because of cache issues). Let's keep that change purely visual for now 🙏

Reply:
ah great point, will make the changes 👍